controllers/serviceimport/reconciler.go

```diff
@@ -43,12 +45,12 @@ type Reconciler struct {
 	Scheme        *runtime.Scheme
 	ClusterID     string
 	EventRecorder *events.EventRecorder
-
 	// metrics
 	gaugeEndpoints *prometheus.GaugeVec
 }
 
 var finalizerName = "networking.kubeslice.io/serviceimport-finalizer"
+var controllerName = "serviceimport_controller"
```

Comment:
Don't we follow naming convention serviceimport_controller Is it camelcase or _ or -
can we be consistent

Reply:
currently some places we are using <name>-controller and some places it is <name>_controller, so we will have to change all occurrences to one single format.